_pages/cfp.md

```diff
@@ -26,17 +26,17 @@ All accepted submissions will be accompanied by a poster presentation. A number
 
 ## Scope and Topics
 
-We invite submissions related to pretrained time series models. The key topics include (but are not limited to):
-
-- Building _foundation_ models for time series data.
-- Analyzing existing pretrained time series models (e.g., explainability and interpretability).
-- Critiques on foundation models for time series.
-- Faster and better inference schemes for autoregressive pretrained time series models.
-- Leveraging pretrained models of other modalities for time series.
-- Multimodal time series models.
-- Large scale evaluation benchmarks, live (continuously updating) datasets, synthetic data generation and data augmentation for time series.
-- Time series evaluation (e.g. analysis and development of metrics for (probabilistic) forecasting, multivariate forecasting, use-case motivated metrics etc.)
-- Real-world applications of large time series models.
+We invite submissions related to time series foundation models (TSFMs). Key topics include, but are not limited to:
+
+- **Building Foundation Models for Time Series Data**: The heterogeneity of time series data and tasks presents unique challenges in developing TSFMs. We welcome contributions exploring various design choices and improving our understanding of how these models scale with the amount and diversity of data.
+- **Analyzing Existing Pretrained Time Series Models**: TSFMs are often criticized for their black-box nature, especially compared to interpretable statistical models. We encourage submissions that analyze pretrained time series models to enhance our understanding of their learning processes.
+- **Critiques of Foundation Models for Time Series**: Contributions highlighting the limitations of TSFMs through theoretical analysis or systematic empirical evaluations are welcome.
```

Comment:
* Critiques `on` foundation models...
* Also, limitations is a nice term, but it'll also be nice to include "failure modes".

Reply:
Fixed.